apps/recipes-react-components/package.json

```diff
@@ -17,7 +17,7 @@
   },
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30078&commitId=d16b15253bf9d39fb43f18de76133b44fb2509af&env=prod&clientType=FLUENTUI)


  <details><summary><b>InfoLabel</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InfoLabel.default - High Contrast.rest.chromium.png|57| Changed|
|InfoLabel.default - Dark Mode.rest.chromium.png|39| Changed|
|InfoLabel.required.rest.chromium.png|43| Changed|
|InfoLabel.default.rest.chromium.png|43| Changed|
|InfoLabel.default - RTL.rest.chromium.png|43| Changed|
|InfoLabel.wrap.rest.chromium.png|43| Changed|
|InfoLabel.required - RTL.rest.chromium.png|43| Changed|
|InfoLabel.sizes -- open.rest.chromium.png|25| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>